pkg/mcs/resourcemanager/server/token_buckets_test.go

```diff
@@ -144,45 +144,46 @@ func TestGroupTokenBucketRequestLoop(t *testing.T) {
 	gtb.Tokens = 50000
 
 	const timeIncrement = 5 * time.Second
-	const defaultTrickleDuration = int64(time.Second) * 5 / int64(time.Millisecond)
+	const targetPeriod = 5 * time.Second
+	const defaultTrickleMs = int64(targetPeriod) / int64(time.Millisecond)
```

Comment:
The `Millisecond` here is meaningless
```suggestion
	const defaultTrickleMs = int64(targetPeriod) / int64(1000)
```